.github/workflows/warn-on-editing-requirements.yml

```
@@ -9,7 +13,9 @@ permissions:
 
 jobs:
   check:
-    if: github.event.review.state == 'APPROVED'
+    if: |
+      github.event_name == 'pull_request_target'
+      || github.event.review.state == 'APPROVED'
```

```suggestion
    if: github.event_name == 'pull_request_target' || github.event.review.state == 'APPROVED'
```

Can we try this? This workflow should run on this PR, but it doesn't.

---

ah let's try `pull_request` first.